src/components/accessRequests/accessRequestsFilter/accessRequestsFilter.tsx

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+// for the German Human Genome-Phenome Archive (GHGA)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFilter } from "@fortawesome/free-solid-svg-icons";
+import { Button, Col, Container, Form, Row } from "react-bootstrap";
+
+interface AccessRequestsFilterProps {
+  handleFilter: any;
+  MIN_YEAR: number;
+  MAX_YEAR: number;
+  parseDate: (value: string) => string;
+}
+
```

Comment:
Consider passing the filter object with the 5 values in as part of the props. The default values would then be those set in `useState<Filter>` in the wrapper component.

Reply:
Done